src/main/java/com/yelp/nrtsearch/server/luceneserver/search/collectors/additional/NestedCollectorManagers.java

```diff
@@ -110,7 +110,7 @@ public Map<String, CollectorResult> reduce(
       Collection<Collector> collectors =
           nestedCollectors.stream()
               .map(m -> m.nestedCollectorsByValue.get(value))
-              .filter(c -> c.containsKey(entry.getKey()))
+              .filter(c -> c != null && c.containsKey(entry.getKey()))
```

Comment:
Will the python None value from the py grpc client translate to null for checking in this case?

Reply:
This value is not user provided, so it shouldn't be an issue.